src/styles/components/buttons/styles.less

```diff
@@ -49,48 +57,45 @@ components/buttons.less
       &.button-fixed-width-small {
         width: @base-spacing-unit * 3;
       }
+
     }
 
     &-up-arrow,
     &-down-arrow {
+
       &:after {
         border-left: 5px solid transparent;
         border-right: 5px solid transparent;
         content: '';
         display: inline-block;
         vertical-align: middle;
       }
+
     }
 
     &-up-arrow {
+
       &:after {
         border-bottom: 6px solid color-lighten(@neutral, 10);
       }
+
     }
 
     &-down-arrow {
+
       &:after {
         border-top: 6px solid color-lighten(@neutral, 10);
       }
-    }
-  }
 
-  .button-leave {
-    opacity: 1;
-    transform: translate(-50%, 0);
-
-    &.button-leave-active {
-      opacity: 0.01;
-      transform: translate(-50%, -100%);
-      transition: transform 350ms ease-in, opacity 350ms ease-in;
     }
+
   }
 
   .button-collection {
 
     & > .dropdown {
```

Comment:
This another one we should circle back around on.  Seems like there is a style-hack happening here as result of either the component being implemented incorrectly in DOM or we need to modify the source `.button-collection` object to fix whatever issue this is trying to solve.  Need more understanding.